hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java

```diff
@@ -45,12 +48,31 @@
 
   public OzoneConfiguration() {
     OzoneConfiguration.activate();
+    loadDefaults();
   }
 
   public OzoneConfiguration(Configuration conf) {
     super(conf);
     //load the configuration from the classloader of the original conf.
     setClassLoader(conf.getClassLoader());
+    loadDefaults();
+  }
+
+  private void loadDefaults() {
```

Comment:
Can I be a greedy pig and request that we also sort these keys (in another JIRA of course) before we write out the XML?